cadcutils/cadcutils/net/auth.py

```diff
@@ -69,25 +69,103 @@
                         unicode_literals)
 
 import getpass
-import netrc
+import netrc as netrclib
 import argparse
 import os
 import signal
 import sys
 import requests
+import logging
 
 CERT_ENDPOINT = "/cred/proxyCert"
 CERT_SERVER = "www.canfar.phys.uvic.ca"
 
-__all__ = ['get_cert', 'get_user_password']
+__all__ = ['get_cert', 'get_user_password', 'Subject']
 
 
-def get_cert(cert_server=None,
+class Subject:
```

Comment:
Subject(object) is the new style classes that we are supposed to use which I've forgotten. Thanks for bringing it to my attention. Fixed.